pytorch_blade/torch_blade/pass_manager.py

```diff
@@ -61,7 +75,7 @@ def _export_onnx(graph, dynamic_axes):
     custom_opsets = {}
     val_add_node_names = True
     proto = graph._export_onnx(
-        dict(),
+        params_dict,
```

Comment:
Will this freeze extra-parameters into the proto as constants?

Reply:
No. `params_dict` should contain frozen constants only. `_jit_pass_onnx_constfold` will return a `params_dict` like that.